### PR TITLE
Disallow marshalling of continuations

### PR DESCRIPTION
**Pull request by vouillon**
I don't think it makes sense to marshal continuations. Indeed, at the moment we are just marshalling a pointer to a `struct stack_info`. This makes it raise an invalid argument exception.

### CODE REVIEW

**Review by stedolan — APPROVED** [review]
Makes sense, well spotted!

(I had a quick look at the other polymorphic operations (comparison & hashing) seem to have reasonable `Cont_tag` cases already)

**Comment by hhugo** [comment]
Will this be back ported to 5.0 ?

**Comment by Octachron** [comment]
Possibly for 5.0.1 if such version ever exists. But the bug is not in scope for 5.0.0 .

**Comment by vouillon** [comment]
> Makes sense, well spotted!

Thanks @hhugo for noticing that something was amiss regarding marshalling!

File: Changes [hunk 1]
```diff
@@ -284,6 +284,9 @@ Working version
 - #11809: Protect Parmatch.pats_of_type from missing cmis
   (Jacques Garrigue, review by Stephen Dolan and Gabriel Scherer)
 
+- #11815: Marshalling continuations raises invalid argument exception.
+  (Jérôme Vouillon)
```
Comment (nojb) [inline Changes]: Please add list of reviewers.

**Review by nojb — COMMENTED** [review]
This can be merged as soon as the conflict in `Changes` is resolved.

File: runtime/extern.c [hunk 2]
```diff
@@ -853,6 +853,9 @@ static void extern_rec(struct caml_extern_state* s, value v)
       v = Field(v, i);
       continue;
     }
+    case Cont_tag: {
+      extern_invalid_argument(s, "output_value: continuation value");
+    }
```
Comment (hhugo) [inline runtime/extern.c]: note that the code style for Abstract_tag is different.
```c
    case Abstract_tag:
      extern_invalid_argument(s, "output_value: abstract value (Abstract)");
      break;
```

Comment (vouillon) [reply runtime/extern.c]: I have updated the PR for consistency

Comment (hhugo) [reply runtime/extern.c]: LGTM

**Comment by nojb** [comment]
CI failure is unrelated.

**Pull request merged**